### PR TITLE
[DT-298][risk=no] Update style, navigation for Tanagra iframe

### PR DESCRIPTION
**Pull request by dolbeew**
- Listen for events from `postMessage` api to hide Tanagra iframe
- Get correct Tanagra url from `serverConfigStore`
- Update iframe styles for a smoother transition to and from the workbench
- Add property `bigqueryDataset` to the `CdrVersion` object in `CdrVersionTiersResponse` to use as the Tanagra underlay

https://github.com/all-of-us/workbench/assets/40036095/908df294-25aa-4011-9fa6-00871d6ab259

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/pages/tanagra-dev/tanagra-dev.tsx [hunk 1]
```diff
@@ -1,30 +1,85 @@
...
+      {loadingIframe && <SpinnerOverlay />}
       <iframe
+        onLoad={() => setLoadingIframe(false)}
         style={{
           border: 0,
           height: '100%',
           width: '100%',
         }}
-        src='http://localhost:3000/#/'
+        src={`${tanagraUrl}/#/tanagra/underlays/SR2022Q4R6/studies/${studyId}/export`}
```
Comment (chenchals) [inline ui/src/app/pages/tanagra-dev/tanagra-dev.tsx:65]: Is there a reason we are hard-coding the underlay to SR2022Q4R6?

File: ui/src/app/pages/tanagra-dev/tanagra-dev.tsx [hunk 2]
```diff
@@ -1,30 +1,85 @@
...
   }, []);
 
-  return (
+  useExitActionListener(() => {
+    setShowIframe(false);
+  });
+
+  // Temporary variable to hardcode existing study ids until we have a link between workspaces and studies
+  const studyId =
+    process.env.REACT_APP_ENVIRONMENT === 'local' ? 'tqmXfZ4qzu' : 'UslvvbIYxk';
```
Comment (chenchals) [inline ui/src/app/pages/tanagra-dev/tanagra-dev.tsx:48]: I thought we will preload the study table with workspaceId for the Id column, then we would not have to save study for the demo.?

Comment (dolbeew) [reply ui/src/app/pages/tanagra-dev/tanagra-dev.tsx:47]: Changed to use the workspace namespace, which will also be the Tanagra study id.

File: ui/src/app/pages/tanagra-dev/tanagra-dev.tsx (hunk 1 above)
Comment (dolbeew) [reply ui/src/app/pages/tanagra-dev/tanagra-dev.tsx:64]: Added `bigqueryDataset` property to the `CdrVersion` object to use here.

File: ui/src/app/pages/tanagra-dev/tanagra-dev.tsx [hunk 3]
```diff
@@ -1,30 +1,76 @@
...
-});
+      >
+        <iframe
+          onLoad={() => hideSpinner()}
+          style={{
+            border: 0,
+            height: '100%',
+            width: '100%',
+          }}
+          src={`${tanagraUrl}/#/tanagra/underlays/${bigqueryDataset}/studies/${namespace}/export`}
```
Comment (chenchals) [inline ui/src/app/pages/tanagra-dev/tanagra-dev.tsx:71]: perfect!

**Review by chenchals — APPROVED** [review]
LGTM! Thank you for taking out hard-coded values!

**Review by freemabd — APPROVED** [review]

**Pull request merged**